.travis.yml

```diff
@@ -6,7 +6,7 @@ cache:
 notifications:
   email: false
 node_js:
-  - '4'
+  - stable
```

Comment:
Maybe use `stable` and `latest`? That way we'd get the current LTS and the current stable release covered.

Shouldn't 6.x have been released and 5.x be considered deprecated on April 1st? I thought so, but I cannot find that roadmap from mobile right now.